app/styles/globals.css

```
@@ -1,6 +1,5 @@
-@import 'tailwindcss';
-
-/* import sub style */
+@import url('https://hangeul.pstatic.net/hangeul_static/css/maru-buri.css');
```

Stylelint (17.4.0)

[error] Expected "url('https://hangeul.pstatic.net/hangeul_static/css/maru-buri.css')" to be "'https://hangeul.pstatic.net/hangeul_static/css/maru-buri.css'" (import-notation)

(import-notation)